google/resource_compute_region_autoscaler.go

```diff
@@ -0,0 +1,372 @@
+package google
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/compute/v1"
+)
+
+func resourceComputeRegionAutoscaler() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceComputeRegionAutoscalerCreate,
+		Read:   resourceComputeRegionAutoscalerRead,
+		Update: resourceComputeRegionAutoscalerUpdate,
+		Delete: resourceComputeRegionAutoscalerDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Required: true,
+			},
+
+			"target": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"region": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"autoscaling_policy": &schema.Schema{
+				Type:     schema.TypeList,
+				Optional: true,
```

Comment:
Add `MaxItems: 1` since we can only specify at most on `autoscaling_policy`. It is also not Optional since you must have at least one `autoscaling_policy`. Please add `Required: true`